stable/oauth2-proxy/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: oauth2-proxy
-version: 0.2.3
+version: 0.2.4
```

Comment:
You're adding a new feature, please bump the minor version.

Reply:
Done. @timstoop